integration_tests/mockApis/licence.ts

```diff
@@ -120,7 +120,7 @@ export default {
             {
               id: 3,
               code: 'a7c57e4e-30fe-4797-9fe7-70a35dbd7b65',
-              category: 'Participation in, or co-operation with, a programme or set of activities',
+              category: 'Programmes or activities',
               sequence: 2,
```

Comment:
For some categories, this is what will be supplied to the API when updating them, so it correct that the mock API should return the shortened names.